toc.md

```diff
@@ -3,7 +3,9 @@
 1. [Basic Data Types and Operators](#basic-data-types)
 2. [Variables](#variables)
 3. [Conditionals](#if)
+4. [Conditional cont.](#lesson4)
+5. [Functions cont.](#functions)
```

Comment:
Cont? It's a new topic
:)

Also there's a slide with the id "#functions" that you can refer to here so students will land directly at the topic, not at the beginning of lesson 5

Reply:
Whoops yes I was reading the wrong line it seems. It's correct. Sorry for the hiccup :)